assets/facets.js

```diff
@@ -1,49 +1,42 @@
 class FacetFiltersForm extends HTMLElement {
   constructor() {
     super();
-    this.filterData = [];
     this.onActiveFilterClick = this.onActiveFilterClick.bind(this);
 
     this.debouncedOnSubmit = debounce((event) => {
       this.onSubmitHandler(event);
     }, 500);
 
     this.querySelector('form').addEventListener('input', this.debouncedOnSubmit.bind(this));
-    window.addEventListener('popstate', this.onHistoryChange.bind(this));
-
+    
     const facetWrapper = this.querySelector('#FacetsWrapperDesktop');
     if (facetWrapper) facetWrapper.addEventListener('keyup', onKeyUpEscape);
   }
 
-  onSubmitHandler(event) {
-    event.preventDefault();
-    const formData = new FormData(event.target.closest('form'));
-    const searchParams = new URLSearchParams(formData).toString();
-    this.renderPage(searchParams, event);
-  }
+  static filterData = [];
+  static searchParamsInitial = window.location.search.slice(1);
+  static searchParamsPrev = window.location.search.slice(1);
```

Comment:
Ah shoot I think we can't use this yet as it's not supported on previous version of Safari when we're supposed to support the last 2 versions: 
![](https://screenshot.click/21-09-f7v9n-duw3r.png)

Reply:
I didn't re test the rest of the functionality as I think we won't be able to use that approach 🙁

Reply:
Ah good point. I will look into an alternative

Reply:
In my testing, it appears `static` methods work fine even on iOS 13, but `static` class properties were causing a JS error. I've therefore moved the properties out of the class.